module-4/backend/week_two.md

```diff
@@ -1,22 +1,57 @@
 ## Week Two - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
 
-Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+  Syntax in various areas.
+  ;'s not required.
+  explicit returns not required. 
```

Comment:
Careful about the explicit returns - that's only true for one-line function bodies